tensorflow/src/TensorFlow/Session.hs

```diff
@@ -135,7 +135,7 @@ extend = do
     trace <- Session (asks tracer)
     nodesToExtend <- build flushNodeBuffer
     unless (null nodesToExtend) $ liftIO $ do
-        let graphDef = def & node .~ nodesToExtend
+        let graphDef = (def :: GraphDef) & node .~ nodesToExtend
```

Comment:
For future reference (no changes needed for this PR): `proto-lens-0.2.*` made this extra type annotation necessary.  However, we can avoid it by switching the tensorflow packages away from `lens-family`.  Any of `lens-family-core`, `microlens` or `lens` would be sufficient.

Some notes on this topic are here:
https://github.com/google/proto-lens/blob/master/proto-lens-tests/tests/labels_test.hs#L40